### PR TITLE
exclude time from default date widget format

### LINKED ISSUE #1119: Date widget in list returning Datetime instead of Date
**- Do you want to request a *feature* or report a *bug*?**

**- What is the current behavior?**

**- If the current behavior is a bug, please provide the steps to reproduce.**

**- What is the expected behavior?**

**- Please mention your CMS, node.js, and operating system version.**

**- Please link or paste your `config.yml` below if applicable.**

I'm using a date widget in a list, but noticed it was returning a datetime stamp instead of a date formatted "YYYY-MM-DD" as described in the docs ([date widget](https://www.netlifycms.org/docs/widgets/#date)). My json file after creating dates from within the CMS:

```
{
"blackout_dates": [
{
"note": "Vacation",
"start_date": "2018-02-19T06:00:00.000Z",
"end_date": "2018-02-23T18:50:33.987Z"
}
]
}
```

config file: [config.yml](https://github.com/thebetternewt/loveleerolls_victor_hugo/blob/master/site/static/admin/config.yml)

For now I have just set a "format" option with the value "YYYY-MM-DD" to enforce the expected output.

### PR DESCRIPTION
**Pull request by erquhart**
Sets correct and documented format for Date widget (YYYY-MM-DD).

Closes #1119.

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 3bb1dc5ca6c76b1e23ee4dd024304fbfdbdf1593

https://deploy-preview-1143--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 3bb1dc5ca6c76b1e23ee4dd024304fbfdbdf1593

https://deploy-preview-1143--cms-demo.netlify.com

File: src/components/EditorWidgets/Date/DateControl.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default class DateControl extends React.Component {
 
   render() {
     const { includeTime, value, classNameWrapper, setActiveStyle, setInactiveStyle } = this.props;
-    const format = this.format || moment.defaultFormat;
+    const format = this.format || moment[includeTime ? 'defaultFormat' : 'HTML5_FMT.DATE'];
```
Comment (tech4him1) [inline src/components/EditorWidgets/Date/DateControl.js]: `'HTML5_FMT.DATE'` won't work as it's not a key name, since `DATE` is a property of `HTML5_FMT`.

Comment (tech4him1) [reply src/components/EditorWidgets/Date/DateControl.js]:
```js
const format = this.format || (includeTime ? moment.defaultFormat : moment.HTML5_FMT.DATE);
```
should work (not tested)

**Comment by tech4him1** [comment]
I'm assuming this shouldn't be classified as a breaking change if we're just changing it to match our docs? It is changing the output of pre-existing entries, though.

File: src/components/EditorWidgets/Date/DateControl.js (hunk 1 above)
Comment (erquhart) [reply src/components/EditorWidgets/Date/DateControl.js]: Good catch!

**Comment by erquhart** [comment]
Right not, breaking, just a bugfix. This is the documented behavior and we weren't doing it.

**Comment by erquhart** [comment]
@tech4him1 fixed.

**Comment by tech4him1** [comment]
@erquhart Code looks great! However, `HTML5_FMT.DATE` wasn't added until Moment 2.20.0, and we're on 2.11.2, so that will need updated as well.

**Comment by erquhart** [comment]
Drat.

**Comment by erquhart** [comment]
@tech4him1 fixed 4 realz.

**Comment by tech4him1** [comment]
@erquhart I went ahead and pushed a commit to save the formatted value in Redux, feel free to revert if you don't like it. 😄

**Comment by erquhart** [comment]
@tech4him1 much better place to do it, thank you.

**Pull request merged**